Training/2.0 Libraries.md

```diff
@@ -51,14 +63,25 @@ Normally we would want to be able to extract the information from tables directl
 
 [Training2_example.csv](https://github.com/alepgr/gubmes-hda/blob/main/Training/Training2_example.csv)
 
-```
+``` import pandas as pd
+``` open("training2data.xlsx")
+``` xl = pd.ExcelFile("training2data.xlsx")
+``` xl.sheet_names
+``` df = xl.parse("Sheet1")
+``` print(pd.DataFrame(df))
```

Comment:
Technically what you've done is correct, but you've cheated a bit by converting the .csv file to an Excel Workbook. You'll find when we start working on the projects that most published data sets (as well as data from most physical instruments) come as .csv files. So being able to work with them directly in Python is important. Correct solution is:
`df = pd.read_csv('Training2_example.csv')`